src/informalsystems.rs

```diff
@@ -521,13 +537,9 @@ impl<T: Config> ClientKeeper for Context<T> {
 	fn increase_client_counter(&mut self) {
 		log::info!("in increase client counter");
 
-		match <Pallet<T> as Store>::ClientCounterV2::get() {
-			None => {}
-			Some(old) => {
-				let new = old.checked_add(1).unwrap();
-				<Pallet<T> as Store>::ClientCounterV2::put(new)
-			}
-		}
+		let old = <Pallet<T> as Store>::ClientCounter::get();
```

Comment:
use mutate or try_mutate here.
see https://github.com/octopus-network/pallet-octopus-appchain/blob/fa9a54ec69d871d521b11c681e77eec5a19051c0/src/lib.rs#L511

Reply:
StorageValue mutate and try_mutate impl: https://github.com/octopus-network/substrate/blob/bab9deca26db20bfc914263e0542a7a1b0d8f174/frame/support/src/storage/generator/value.rs#L107

relate issue: https://github.com/paritytech/substrate/pull/5699/files#